lib/service/database.js

```diff
@@ -68,26 +68,41 @@ var affectedTableRegexCache = {
 
 DatabaseService.prototype.getLastUpdatedTimeFromAffectedTables = function(node, skip, callback) {
     if (!!skip || node.type !== Source.TYPE || node.getUpdatedAt() !== null) {
-        return callback(null, node.getUpdatedAt());
+        var data = {'last_update': node.getUpdatedAt(),
+                    'affected_tables': node.getAffectedTables()};
+        return callback(null, data);
     }
 
     var sql = node.sql()
         .replace(affectedTableRegexCache.bbox, 'ST_MakeEnvelope(0,0,0,0)')
         .replace(affectedTableRegexCache.scaleDenominator, '0')
         .replace(affectedTableRegexCache.pixelWidth, '1')
         .replace(affectedTableRegexCache.pixelHeight, '1');
-    sql = 'SELECT max(updated_at) FROM CDB_QueryTables_Updated_At($camshaft$' + sql + '$camshaft$)';
+    sql = 'SELECT max(updated_at), schema_name, table_name ' +
```

Comment:
Hum I was doing that to remove duplicated for the same table but looking at it now seems like that is not possible, so removing group and agg function.